app/classifier/tasks/transcription/SubtaskEditor.jsx

```diff
@@ -62,6 +62,21 @@ export default function SubTaskEditor({ subtask, subtaskPrefix, workflow }) {
           <input type="checkbox" value="unclear" checked={tagExists('unclear')} onChange={updateTags} />
           {"Unclear"}
         </label>
+        <br/>
+        <label>
+          <input type="checkbox" value="underline" checked={tagExists('underline')} onChange={updateTags} />
+          {"Underline"}
+        </label>
+        <br/>
+        <label>
+          <input type="checkbox" value="superscript" checked={tagExists('superscript')} onChange={updateTags} />
+          {"Superscript"}
+        </label>
+        <br/>
+        <label>
+          <input type="checkbox" value="&amp;" checked={tagExists('&')} onChange={updateTags} />
+          {"&"}
```

Comment:
Does JSX automatically escape entities in strings but not in attributes? That's confusing.

Reply:
Yep, that's JSX for ya - if the code ain't in `{javascript_escape_brackets}`, you're writing the value in ol' HTML. I could have made it less confusing by writing `value={'&'}` but that's still a bit chonky.

Reply:
Yeah, I had to stop and think about that for a second. I did see that the code sets the input's ID to something like `T0.0.0-&` in the DOM, but the browser has no problems parsing that and it didn't break the accessible label so 🤷‍♂️ .